sprints/2024/02_08/margin_crossmatch.ipynb

```diff
@@ -0,0 +1,542 @@
+{
```

Comment:
<p>Same here, and for the remaining plot calls :(</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/notebooks_lf/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>